plugins/webp-uploads/helper.php

```diff
@@ -395,3 +395,14 @@ function webp_uploads_sanitize_image_format( string $image_format ): string {
 function webp_uploads_is_picture_element_enabled(): bool {
 	return (bool) get_option( 'webp_uploads_use_picture_element', false );
 }
+
+/**
+ * Checks if the `perflab_generate_webp_and_jpeg` option is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the option is enabled, false otherwise.
+ */
+function webp_uploads_is_generate_webp_and_jpeg_enabled(): bool {
```

Comment:
Maybe this should be named to take into account AVIF now? Maybe like:

```suggestion
function webp_uploads_is_jpeg_fallback_enabled(): bool {
```

Reply:
sure, that makes sense. we just can't easily change the option name.